workshops/popup_chrome_extension/README.md

```diff
@@ -0,0 +1,498 @@
+  - Using [Bootstrap's CSS](https://getbootstrap.com/docs/3.4/css/) library
+  - Added complex [event listeners](https://www.w3schools.com/js/js_htmldom_eventlistener.asp) and tools to recognize button clicks and run API requests
+
+You will be making a Chrome extension that allows you to save and access your Zoom links for your classes. This is a very simple app, but it will allow you to "hack your way" to productivity by teaching you the foundation of building a Chrome extension.
+
+**Difficulty/interest level:** This is a simple web app using HTML/CSS/JS, with a strong emphasis on Javascript. I will go over everything you need to know for this workshop, so you don't need to worry about how much experience you have. This demo will *specifically* teach you how to build a bookmarking application (that is tailored for [Zoom](https://zoom.us/) links), but if you want to learn how to build any type of Chrome extension, this is your workshop!
+
+**Average time for completion:** You can complete this in <u>15-20 minutes</u>. You will need the following tools:
+
+- [Repl.it](https://repl.it/) online coding platform (no account necessary, but highly recommended)
+- [Bootstrap CSS](https://getbootstrap.com/docs/4.4/getting-started/introduction/)
+- [Chrome API](https://developer.chrome.com/extensions/devguide)
+
+**Demo:** Test out what we'll be making by tinkering with this live demo!
+
+You can download/clone this Github repository for the final code, if you want to reference from a master copy: https://github.com/shreyfirst/zoom_bookmarks
+
+Link to download the published version: https://chrome.google.com/webstore/detail/zoom-bookmarks/haplbihlmhnffbihdfjhgnickameiphd?hl=en
+
+![Demo of what the extension will look like](https://cloud-qyx3m9oc6.vercel.app/0image.png)
+
+## Table of Contents
+
+There are a few parts to this workshop. I highly recommend doing them in order because each one relies on the other.
+
+1. Setting up the project
+2. Frontend
+   * Creating the design and importing different libraries/stylesheets to enhance the visual aspect of the app
+   * Adding form fields and linking the data in the code, so we can reference it in the backend later
+
+3. Backend
+   * Plan out what the structure of the app is going to look like
+   * Create different functions to handle different events that could happen
+4. Testing the extension
+
+## Setting up the project
+
+1. Go to to [Repl.it](https://repl.it/) and either:
+
+   1. Create an account
+   2. Click the "Start coding" button on the top right
+
+   **The website will look a little like this:**
+
+   ![Repl.it website)@2x](https://cloud-cnenr6ov1.vercel.app/0image.png)
+
+2. Choose "HTML/CSS/JS"
+
+   * This means that we will be coding in different web languages. HTML is like the text files, CSS is similar to the design element, and JS is anything special like animations, etc
+
+3. Once you have a new HTML Repl, you want to configure it with this file structure:
+
+```
```

Comment:
This may be confusing to beginners. Could you maybe include a gif or something demonstrating how to create all these files and folders?